src/WebhookCall.php

```diff
@@ -31,6 +32,7 @@ public static function create(): self
         $config = config('webhook-server');
 
         return (new static())
+            ->useJob($config['webhook_job'] ?? CallWebhookJob::class)
```

Comment:
Added as a fallback in case user update the dependency without updating the configuration.

Reply:
The fallback isn't needed as the value will be provided by the config.

Reply:
Done :)